src/TrelloManager.py

```diff
@@ -0,0 +1,180 @@
+        load_dotenv()
+        self.api_key = os.getenv("TRELLO_API_KEY")
+        self.token = os.getenv("TRELLO_OAUTH_TOKEN")
+        self.BASE_URL = "https://api.trello.com/1"
+
+    def get_full_board_info(self, short_board_id: str) -> str:
+        """Get full board information from Trello.
+
+        Args:
+            short_board_id (str): The short ID of the Trello board.
+
+        Returns:
+            str: The full board ID.
+
+        """
+        url = f"{self.BASE_URL}/boards/{short_board_id}"
+        query = {
+            "key": self.api_key,
+            "token": self.token,
+        }
+        response = requests.get(url, params=query, timeout=10)
+
+        if response.status_code == HTTP_OK:
+            board_info = response.json()
+            return board_info["id"]
+        return ""
+
+    def get_lists(self, board_id: str) -> list:
+        """Get all lists in the specified Trello board.
+
+        Args:
+            board_id (str): The full ID of the Trello board.
+
+        Returns:
+            list: A list of list IDs.
+
+        """
+        url = f"{self.BASE_URL}/boards/{board_id}/lists"
+        query = {
+            "key": self.api_key,
+            "token": self.token,
+        }
+        response = requests.get(url, params=query, timeout=10)
+
+        list_id = []
+        if response.status_code == HTTP_OK:
+            lists = response.json()
+            list_id = [lst["id"] for lst in lists]
+
+        return list_id
+
+    def create_card(self, list_id: str, name: str, desc: str) -> None:
+        """Create a new card in the specified list.
+
+        Args:
+            list_id (str): The ID of the list where the card will be created.
+            name (str): The name of the card.
+            desc (str): The description of the card (optional).
+
+        """
+        url = f"{self.BASE_URL}/cards"
+        query = {
+            "key": self.api_key,
+            "token": self.token,
+            "idList": list_id,
+            "name": name,
+            "desc": desc,
+        }
+        response = requests.post(url, params=query, timeout=10)
+
+        if response.status_code == HTTP_OK:
+            logger.info("Card successfully created!")
+
+    def get_list_name(self, list_id: str) -> str:
+        """Get the list name of a specific list id in the specific board.
+
+        Args:
+            list_id (str): the id of the specific list.
+
+        Returns:
+            str: the name of the specific list.
+
+        """
+        url = f"{self.BASE_URL}/lists/{list_id}"
+        query = {
+            "key": self.api_key,
+            "token": self.token,
+        }
+        response = requests.get(url, params=query, timeout=10)
+
+        if response.status_code == HTTP_OK:
+            list_info = response.json()
+            return list_info.get("name", "No name found")
+
+        return ""
+
+    def create_a_list(self, list_name: str, id_board: str) -> str:
```

Comment:
Is `create_list()` taken? Generally articles are not included in function names.